### PR TITLE
Enumerate each package on a separate line

### PR DESCRIPTION
**Pull request by jakirkham**
When writing out `pkg_versions.json`, write each one to a separate line. This should keep the diffs cleaner and easier to read.

Also write the commit message so that package versions are in a bullet point list in the body of the message. This will make it easier to follow what was changed and keep the commit messages more readable.

<hr>

Checklist
* [x] Pushed the branch to main repo
* [ ] CI passed on the branch

### CODE REVIEW

File: conda_forge_webservices/update_me.py [hunk 1]
```diff
@@ -98,10 +98,15 @@ def main():
...
             pth = os.path.join(clone_dir, "pkg_versions.json")
             with open(pth, "w") as fp:
-                json.dump(final_install, fp)
+                json.dump(final_install, fp, indent=2)
+                fp.write("\n")
             repo.index.add(pth)
 
-            msg_vers = ", ".join(["{}={}".format(k, v) for k, v in to_install.items()])
+            msg_vers = (
+                "Redeploy for package updates:\n\n" +
```
Comment (isuruf) [inline conda_forge_webservices/update_me.py:106]: Duplicate here

Comment (jakirkham) [reply conda_forge_webservices/update_me.py:106]: Meaning 2 newlines?

That is intentional to include a blank line between the title and body of the commit message in line with the commonly adopted [50/72 commit practice]( https://preslav.me/2015/02/21/what-s-with-the-50-72-rule/ )

Comment (isuruf) [reply conda_forge_webservices/update_me.py:106]: The commit msg now looks like `redeploy for 'Redeploy for package updates: ...'`

Comment (jakirkham) [reply conda_forge_webservices/update_me.py:106]: Thanks for clarifying! 🙏

That was too terse to follow before

Please see my comment below

File: conda_forge_webservices/update_me.py [hunk 2]
```diff
@@ -98,10 +98,15 @@ def main():
...
+                "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
+            )
+
             repo.index.commit("redeploy for '%s'" % msg_vers)
```
Comment (jakirkham) [inline conda_forge_webservices/update_me.py:110]: So do we just want this?

```suggestion
            repo.index.commit(msg_vers)
```

**Comment by beckermr** [comment]
Sorry I merged too fast here! Happy to help in a followup PR.

File: conda_forge_webservices/update_me.py (hunk 2 above)
Comment (isuruf) [reply conda_forge_webservices/update_me.py:110]: No, the current commit message [redeploy for 'conda=24.3.0'](https://github.com/conda-forge/conda-forge-webservices/commit/bd36e1e228c177bff49efdfeb5603d7757bbb16e) is good enough.

Comment (jakirkham) [reply conda_forge_webservices/update_me.py:110]: However when multiple things are updated like commit ( https://github.com/conda-forge/conda-forge-webservices/commit/8ccbbe9d2beaa6fabeda23a4b2a5b6f72f9e34d9 ), it gets a bit verby

Am happy to simplify, but don't agree with reversion

Comment (isuruf) [reply conda_forge_webservices/update_me.py:110]: That rarely happens.

**Pull request merged**